chapters/creating_extension.md

```diff
@@ -117,19 +117,52 @@ For example, there is a list of `BuiltIn` enums, and its `PointSize` entry is:
   },
 ```
 
-An enum entry can also have an `extensions` attribute to list the SPIR-V extensions that enable the feature. As a design principle, we only use these on the capabilities introduced by the extension. Other tokens introduced by the extension are in turn guarded by the corresponding capability.
+Enum entries within a list should be ordered by their value. The spec generator writes table entries in the same order as JSON file order.
```

Comment:
Entries should be ordered by their enumerant value to make it easier to merge multiple versions together (from Khronos, public, and multiple PRs).

Delete the part of the spec generator, it's not true.